interface/admin.py

```diff
@@ -1,6 +1,44 @@
+from tarfile import TarFile
+from tempfile import TemporaryDirectory
+from pathlib import Path
+
 from django.contrib import admin
+from django.http import HttpResponse
+
 from interface.models import Course, Assignment, Submission
 
 admin.site.register(Course)
 admin.site.register(Submission)
-admin.site.register(Assignment)
+
+
+@admin.register(Assignment)
+class AssignmentAdmin(admin.ModelAdmin):
+    actions = ['download_submissions']
+
+    def download_submissions(self, request, _assignment):
+        assignment = _assignment[0]
+        ta_user = request.user
+        ta_group = ta_user.groups.all()[0].name
+
+        submissions = Submission.objects.filter(
+                            assignment=assignment,
+                            user__groups__name=ta_group).exclude(user=ta_user)
```

Comment:
Don't worry about excluding the user's own submission. Let the TAs figure this out on their own. Otherwise you get into questions like "what happens if a user is TA for a course and plain student for another course?"